### PR TITLE
es: Update browser compat/spec sections (part 4)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates some of the browser compatibility data and specification sections for the Spanish locale. Part of work for #11594.

Note: this may also include other minor updates, such as removal of old sections or other small bits of cleanup.

### CODE REVIEW

File: files/es/web/css/-webkit-mask-position-y/index.md [hunk 1]
```diff
@@ -69,11 +69,11 @@ La propiedad CSS `-webkit-mask-position-y` fija la posición inicial vertical de
 
 ## Especificaciones
 
-Not part of any specification.
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/css/-webkit-mask-position-y/index.md:72]: As `en-US`

```suggestion
No forma parte de ninguna especificación.
```

File: files/es/web/css/-webkit-mask-box-image/index.md [hunk 2]
```diff
@@ -64,9 +64,13 @@ Where:
 }
 ```
 
-## Compatibilidad con los distintos navegadores
+## Especificaciones
 
-{{Compat("css.properties.-webkit-mask-box-image")}}
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/css/-webkit-mask-box-image/index.md:69]: As `en-US`

```suggestion
No forma parte de ningún estándar.
```

File: files/es/web/css/-webkit-mask-attachment/index.md [hunk 3]
```diff
@@ -31,9 +31,13 @@ body {
 }
 ```
 
-## Compatibilidad con los distintos navegadores
+## Especificaciones
 
-{{Compat("css.properties.-webkit-mask-attachment")}}
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/css/-webkit-mask-attachment/index.md:36]: As `en-US`

```suggestion
No forma parte de ningún estándar.
```

**Comment by queengooborg** [comment]
Hey @Graywolf9, may I ask that you could squash your individual file/change reviews into one single review?  I get emails/notifications per every single review.  🙃

**Comment by queengooborg** [comment]
Thank you, much appreciated!

File: files/es/web/api/webvr_api/index.md [hunk 4]
```diff
@@ -107,11 +107,13 @@ You can find a number of examples at these locations:
 
 ## Especificaciones
 
-{{Specifications}}
+Esta API se especificó en la antigua [API de WebVR](https://immersive-web.github.io/webvr/spec/1.1/) que fue reemplazada por la [API del dispositivo WebXR](https://immersive-web.github.io/webxr/). Ya no está en camino de convertirse en un estándar.
 
-## Browser compatibility
+Hasta que todos los navegadores hayan implementado las nuevas [API de WebXR](/es/docs/Web/API/WebXR_Device_API/Fundamentals), se recomienda confiar en _frameworks_, como  [A-Frame](https://aframe.io/), [Babylon.js](https://www.babylonjs.com/), o [Three.js](https://threejs.org/), o un [polyfill](https://github.com/immersive-web/webxr-polyfill), para desarrollar aplicaciones WebXR que funcionen en todos los navegadores [\[1\]](https://developer.oculus.com/documentation/web/port-vr-xr/).
```
Comment (Graywolf9) [inline files/es/web/api/webvr_api/index.md:112]:
```suggestion
Hasta que todos los navegadores hayan implementado las nuevas [API de WebXR](/es/docs/Web/API/WebXR_Device_API/Fundamentals), se recomienda confiar en _frameworks_, como [A-Frame](https://aframe.io/), [Babylon.js](https://www.babylonjs.com/), o [Three.js](https://threejs.org/), o un [polyfill](https://github.com/immersive-web/webxr-polyfill), para desarrollar aplicaciones WebXR que funcionen en todos los navegadores [\[1\]](https://developer.oculus.com/documentation/web/port-vr-xr/).
```

**Review by Graywolf9 — APPROVED** [review]
Thank you for the PR!

**Pull request merged**